### PR TITLE
Splitter: change focusable element 

### PR DESCRIPTION
**Pull request by nikkithelegendarypokemonster**

### CODE REVIEW

File: apps/demos/Demos/Splitter/Overview/React/App.tsx [hunk 1]
```diff
@@ -1,15 +1,22 @@
-import React from 'react';
+import React, { useEffect } from 'react';
```
Comment (EugeniyKiyashko) [inline apps/demos/Demos/Splitter/Overview/React/App.tsx]: useEffect is unused here. Could you please also check eslint fixable errors for all approaches after your changes?

File: apps/demos/Demos/Splitter/Overview/Vue/App.vue [hunk 2]
```diff
@@ -38,6 +42,7 @@
               min-size="5%"
               text="Nested Right Pane"
               template="paneContent"
+      tabindex="0"
```
Comment (EugeniyKiyashko) [inline apps/demos/Demos/Splitter/Overview/Vue/App.vue]: It should be formatting issue. Repair formatting please

Comment (nikkithelegendarypokemonster) [reply apps/demos/Demos/Splitter/Overview/Vue/App.vue]: [6d9726b](https://github.com/DevExpress/DevExtreme/pull/27306/commits/6d9726b5c1f6e803a0e4b8aac939c51b5a3e9d2a)

File: apps/demos/Demos/Splitter/Overview/React/App.tsx (hunk 1 above)
Comment (nikkithelegendarypokemonster) [reply apps/demos/Demos/Splitter/Overview/React/App.tsx]: [6d9726b](https://github.com/DevExpress/DevExtreme/pull/27306/commits/6d9726b5c1f6e803a0e4b8aac939c51b5a3e9d2a)

File: apps/demos/Demos/Splitter/Overview/React/App.tsx [hunk 3]
```diff
@@ -1,12 +1,21 @@
...
 import Splitter, { Item } from 'devextreme-react/splitter';
 import PaneContent from './PaneContent.tsx';
 
-const PaneContentWithTitleRender = (title, initialSize?: string) => {
-  const PaneContentRender = (data) => (<PaneContent title={title} {...data} size={initialSize} />);
-  return PaneContentRender;
+const generate = (data,element,title, initialSize?: string) => {
+  const container = document.createElement('div');
+  ReactDOM.render(<PaneContent title={title} {...data} size={initialSize} />, container);
+  console.log(element);
```
Comment (EugeniyKiyashko) [inline apps/demos/Demos/Splitter/Overview/React/App.tsx]: Don't forget remove this

File: apps/demos/Demos/Splitter/Overview/React/App.tsx [hunk 4]
```diff
@@ -1,12 +1,21 @@
 import React from 'react';
+import ReactDOM from 'react-dom';
 import Splitter, { Item } from 'devextreme-react/splitter';
 import PaneContent from './PaneContent.tsx';
 
-const PaneContentWithTitleRender = (title, initialSize?: string) => {
-  const PaneContentRender = (data) => (<PaneContent title={title} {...data} size={initialSize} />);
-  return PaneContentRender;
+const generate = (data,element,title, initialSize?: string) => {
```
Comment (EugeniyKiyashko) [inline apps/demos/Demos/Splitter/Overview/React/App.tsx]: I think that spaces should be presented here. Don't forget run `eslint --fix` command

File: apps/demos/testing/widgets/splitter/Overview.test.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+const PANE_CONTENT_CLASS = 'dx-splitter-item-content';
+
+fixture('Splitter.Overview')
+  .page('http://localhost:8080/')
+  .before(async (ctx) => {
+    ctx.initialWindowSize = [900, 800];
+  });
+
+  runManualTest('Splitter', 'Overview', ['jQuery', 'React', 'Vue', 'Angular'], (test) => {
```
Comment (ksercs) [inline apps/demos/testing/widgets/splitter/Overview.test.js]: Please fix incorrect indents and add an empty line in the end of the file

File: apps/demos/Demos/Splitter/Overview/React/App.tsx [hunk 6]
```diff
@@ -15,7 +24,7 @@ const App = () => (
         resizable={true}
         size="140px"
         minSize="70px"
-        render={PaneContentWithTitleRender('Left Pane', '140px')}
+        template={(data,index,element) => generate(data,element,'Left Pane', '140px')}
```
Comment (ksercs) [inline apps/demos/Demos/Splitter/Overview/React/App.tsx]: we cannot use `template` property here, it's just not a React-way. In React we are supposed to use only `component` or `render` options, but both of it does not contain `element` as a parameter.

Seems like our goal can be achieved only using `onItemRendered`

**Review by ksercs — APPROVED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

File: apps/demos/Demos/Splitter/Overview/React/App.tsx (hunk 6 above)
Comment (nikkithelegendarypokemonster) [reply apps/demos/Demos/Splitter/Overview/React/App.tsx]: [80ddd95](https://github.com/DevExpress/DevExtreme/pull/27306/commits/80ddd957e53091fd0528eccd3f69be639215fa14)

File: apps/demos/Demos/Splitter/Overview/React/App.tsx (hunk 4 above)
Comment (nikkithelegendarypokemonster) [reply apps/demos/Demos/Splitter/Overview/React/App.tsx]: [da2d6de](https://github.com/DevExpress/DevExtreme/pull/27306/commits/da2d6de39f6fe219b3290ed0515b53976a732d19)

File: apps/demos/Demos/Splitter/Overview/React/App.tsx (hunk 3 above)
Comment (nikkithelegendarypokemonster) [reply apps/demos/Demos/Splitter/Overview/React/App.tsx]: [da2d6de](https://github.com/DevExpress/DevExtreme/pull/27306/commits/da2d6de39f6fe219b3290ed0515b53976a732d19)

**Pull request merged**